akvo/rsr/static/tests/scripts-src/my-results/utils.test.js

```diff
@@ -16,28 +16,14 @@ import { mock } from "../../../scripts-src/my-results/store";
 const middlewares = [];
 const mockStore = configureStore(middlewares);
 
-let expect = chai.expect;
+// This allows us to test non-exported functions
+// See: https://medium.com/@macsikora/how-to-test-private-functions-of-es6-module-fb8c1345b25f
+const rewire = require('rewire')
```

Comment:
I think it's possible to import rewire in the same manner as all other libraries:
```javascript
import rewire from "rewire";
```